background/setBackground.js

```diff
@@ -0,0 +1,5 @@
+const setBackground = (weatherIcon) => {
+	const background = document.getElementsByClassName("background")[0];
```

Comment:
I think it would be better to go with id here. You could set a lot of elements to have class='background'.
If you have something unique, go with id or name (name preferable for inputs)

Reply:
Done